scalafix-tests/input/src/main/scala/test/escapeHatch/AnnotationScopes.scala

```diff
@@ -0,0 +1,273 @@
+    def this(eDummy: String, fDummy: String) {
+      this(eDummy.toInt, fDummy.toInt)
+    }
+
+    def gDummy( // assert: EscapeHatchDummyLinter
+                hDummy: Int, // assert: EscapeHatchDummyLinter
+                iDummy: Int): Unit = { // assert: EscapeHatchDummyLinter
+      val jDummy = 0 // assert: EscapeHatchDummyLinter
+      var hDummy = 0 // assert: EscapeHatchDummyLinter
+    }
+  }
+
+  // Secondary constructor parameter
+  class DummyClass_6( // assert: EscapeHatchDummyLinter
+                 val aDummy: Int, // assert: EscapeHatchDummyLinter
+                 val bDummy: Int) { // assert: EscapeHatchDummyLinter
+
+    val cDummy = 0 // assert: EscapeHatchDummyLinter
+    var dDummy = 0 // assert: EscapeHatchDummyLinter
+
+    def this(@SuppressWarnings(Array("EscapeHatchDummyLinter")) eDummy: String,
+             fDummy: String) { // assert: EscapeHatchDummyLinter
+      this(eDummy.toInt, fDummy.toInt) // assert: EscapeHatchDummyLinter
+    }
+
+    def gDummy( // assert: EscapeHatchDummyLinter
+                hDummy: Int, // assert: EscapeHatchDummyLinter
+                iDummy: Int): Unit = { // assert: EscapeHatchDummyLinter
+      val jDummy = 0 // assert: EscapeHatchDummyLinter
+      var hDummy = 0 // assert: EscapeHatchDummyLinter
+    }
+  }
+
+  // Method
+  class DummyClass_7( // assert: EscapeHatchDummyLinter
+                 val aDummy: Int, // assert: EscapeHatchDummyLinter
+                 val bDummy: Int) { // assert: EscapeHatchDummyLinter
+
+    val cDummy = 0 // assert: EscapeHatchDummyLinter
+    var dDummy = 0 // assert: EscapeHatchDummyLinter
+
+    def this(eDummy: String, // assert: EscapeHatchDummyLinter
+             fDummy: String) { // assert: EscapeHatchDummyLinter
+      this(eDummy.toInt, fDummy.toInt) // assert: EscapeHatchDummyLinter
+    }
+
+    @SuppressWarnings(Array("EscapeHatchDummyLinter"))
+    def gDummy(hDummy: Int, iDummy: Int): Unit = {
+      val jDummy = 0
+      var hDummy = 0
+    }
+  }
+
+  // Method parameter
+  class DummyClass_8( // assert: EscapeHatchDummyLinter
+                 val aDummy: Int, // assert: EscapeHatchDummyLinter
+                 val bDummy: Int) { // assert: EscapeHatchDummyLinter
+
+    val cDummy = 0 // assert: EscapeHatchDummyLinter
+    var dDummy = 0 // assert: EscapeHatchDummyLinter
+
+    def this(eDummy: String, // assert: EscapeHatchDummyLinter
+             fDummy: String) { // assert: EscapeHatchDummyLinter
+      this(eDummy.toInt, fDummy.toInt) // assert: EscapeHatchDummyLinter
+    }
+
+    def gDummy( // assert: EscapeHatchDummyLinter
+                @SuppressWarnings(Array("EscapeHatchDummyLinter")) hDummy: Int,
+                iDummy: Int): Unit = { // assert: EscapeHatchDummyLinter
+      val jDummy = 0 // assert: EscapeHatchDummyLinter
+      var hDummy = 0 // assert: EscapeHatchDummyLinter
+    }
+  }
+
+  // Local variable
+  class DummyClass_9( // assert: EscapeHatchDummyLinter
```

Comment:
Ditto. You can assume the position of the annotation enclosing jDummy below is correct, otherwise it would be a scalameta bug.

Reply:
done